cfpb/cfgov-refresh
# Implement the first Ask CFPB browser test

Gherkin features have been written for many things that do not yet have actual Cucumber step definitions. This implements the first scenario for the Ask CFPB feature file.

The scenario was rewritten to not test Python-based redirects that are better tested with Python unit tests. It now tests an old Ask CFPB printable URL pattern that is [redirected via Apache](https://github.com/cfpb/cfgov-refresh/blob/96f1176363ea99b306161bad20191be2755a4798/cfgov/apache/conf.d/redirects.conf#L45).

Two new generic step definitions are added for testing a redirecting URL.

Addresses https://GHE/CFGOV/platform/issues/3715


## Additions

- Two new generic step definitions are added for testing a redirecting URL

## Changes

- Updates first Ask CFPB browser testing scenario to test old redirecting print URLs

## Testing

1. Pull branch
1. `gulp test:acceptance:new --specs=ask_cfpb.feature`
1. See that 1 scenario and 2 steps are passing, the rest are undefined, and there are no failures

## Checklist
